storage/src/test/java/tech/pegasys/artemis/storage/MapDbDatabaseTest.java

```diff
@@ -118,7 +116,7 @@ public void shouldRecreateOriginalGenesisStore() {
 
   @Test
   public void shouldGetHotBlockByRoot() {
-    final Transaction transaction = store.startTransaction(databaseTransactionPrecommit);
+    final Transaction transaction = store.startTransaction(new StubDiskUpdateChannel());
```

Comment:
Slightly odd that we're using `StubDiskUpdateChannel` here but a mock in a lot of other places.  Seems like we could just be using the stub.

Reply:
It's because this is not actually a stub (I named it incorrectly) but an interface that handles writing to actual disk without requiring ChainStorageServer class in the process. I'll change the naming and make this more visible.

Reply:
It also writes to the updateResults list in the process, which is a requirement for some of the tests.

Reply:
Done.